src/bsp/src/posix/src/InterlocManager.cpp

```diff
@@ -123,6 +132,14 @@ void InterlocManager::gazeboUpdateCallback(const gazebo_msgs::ModelStates& msg)
 
         tf2::Transform relTransform = currentAgentTf.inverseTimes(distantAgentTf);
 
+        if (!isTfValid(relTransform)) {
```

Comment:
So if the transform doesn't exist, we will log at every interloc callback (10 Hz). Could we just do it a few times and then just give up on the interloc for this agent?